db/migrate/20241107011000_add_position_to_images.rb

```diff
@@ -0,0 +1,8 @@
+class AddPositionToImages < ActiveRecord::Migration[7.2]
+  def change
+    add_column :images, :position, :integer
+    Image.heal_position_column! created_at: :asc
```

Comment:
fills in the position based on `created_at` timestamp

Reply:
Cool